commons/addons/fs_buckets.md

```diff
@@ -41,14 +41,28 @@ You will be able to retrieve generated data between two deployments.
 </tr>
 <tr>
 <td class="cc-col__price "><span class="label cc-label__price label-info">S</span></td>
-<td>UNLIMITED</td>
+<td>1 GB</td>
```

Comment:
I'm not sure about this change. It makes me think that we only allow 1GB of total space, maybe it should stay like it was